eras/conway/impl/src/Cardano/Ledger/Conway/TxOut.hs

```diff
@@ -9,7 +9,9 @@
 {-# LANGUAGE UndecidableInstances #-}
 {-# OPTIONS_GHC -Wno-orphans #-}
 
-module Cardano.Ledger.Conway.TxOut () where
+module Cardano.Ledger.Conway.TxOut (
+  BabbageTxOut (..),
```

Comment:
We could update CHANGELOG with something like 
```
Exposed Cardano.Ledger.Conway.TxOut` module and `BabbageTxOut` from it 
```

Reply:
Why is this re-export even necessary?